guides/installation/requirements.md

```diff
@@ -0,0 +1,182 @@
+# Requirements
+
+Before installing Shopware 6, you should take a quick look at the requirements to check if your local environment is capable of running it.
+
+## Operating System
+
+Shopware 6 is currently only supported on linux and macOS setups.
+Windows is only supported inside wsl.
+
+## Versions
+
+You can use these commands to check your actual environment:
+
+* `php -v`: Show CLI PHP version
+* `php -m`: Show CLI PHP modules
+* `php -i | grep memory_limit`: Show your actual CLI PHP memory limit
+* `composer -v`: Show your actual composer version
+* `node -v`: Show you actual Node version
+* `npm -v`: Show you actual NPM version
+
+**PHP**
+
+* Tested on 7.4.3, 8.0 and 8.1
+* `memory_limit` 512M minimum
+* `max_execution_time` 30 seconds minimum
+* Extensions:
+  * ext-curl
+  * ext-dom  
+  * ext-fileinfo  
+  * ext-gd  
+  * ext-iconv  
+  * ext-intl  
+  * ext-json  
+  * ext-libxml  
+  * ext-mbstring  
+  * ext-openssl  
+  * ext-pcre  
+  * ext-pdo  
+  * ext-pdo\_mysql  
+  * ext-phar  
+  * ext-simplexml
+  * ext-sodium
+  * ext-xml  
+  * ext-zip  
+  * ext-zlib
+* Composer 2.0 or higher
+
+**SQL**
+
+* Tested on MySQL 5.7.21, and 8.0
+  * Only MySQL 8.0.20 in specific, is not compatible
```

Comment:
Maybe makes sense to create a warning block here

Reply:
Or a table with ✅ and 🚫 for easier visualization 🤔

Reply:
I created a problematic version list.